navigation/tracking/scripts/pdaf.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+
+"""
+Single object tracking
+
+Sub tasks: 
+
+    Use a kalam gain so that the filter is numerically stable. 
+    SINGULAR MATRICIES
+
+    TEST
+        - Visualize position estimates with clutter.
+        - Visualize velocity estimates with vectors. 
+        - Visualize validation gate. 
+        - Does the frequency of "zero detections within gate" correspond with p of detection? 
+
+    Observations: 
+        P pri and post are not symmetrical.
+        P post grows to values over 1000.
+"""
+
+
+class PDAF:
+    def __init__(self):
+        # x = [x, y, x', y']
+
+        self.time_step = 0.1
```

Comment:
In a real tracking pipeline, the timestep will not be constant, and instead depend on factors outside this node such as sensor config and group delay. However, I am unsure if the A matrix for the CV model should be updated appropriately, or if this is a hyperparameter for the model.

This should be looked into 😄

Reply:
It might be sufficient to have a constant time step, and up the process noise. 
Otherwise, I can update the A matrix for every loop.